### PR TITLE
Milestone: 1 Portfolio Mobile View

### PR DESCRIPTION
**Pull request by SimonGideon**
## Parse a Figma Portfolio design to create a UI
In this milestone we:
- Created the first two sections of the portfolio.
- Added a more appealing background image and icons
- Initiated flex boxes to place items.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Simon Gideon</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <img class="bg-image" src="images/header-shapes-mobile.svg" alt="background">
```
Comment (ishimwezachee) [inline Index.html:14]: - [ ] Your background image should be in your CSS so that it cover the whole section well and properly

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        </div>
+    </header>
+    <section>
+        <div class="container2">
+            <div class="hero-text">
+                <h2>I'm Simon Gideon Glad to see you!</h2>
+            </div>
+            <div class="primary-text2">
+                <h4>I'm software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to reach me out.
+                </h4>
```
Comment (ishimwezachee) [inline Index.html:32]: - [ ]  You should add this section in `P` tag as it is mentioned in the requirement of this milestone.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @SimonGideon,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.

## Required Changes ♻️

 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [ ] Kindly capitalize the first letter of your commit message in the next milestone so it looks professional.
 - [ ] Kindly work on the styling
 Figma Design
Capturefigrace
Your Design
Capturesimeon



 _Check the comments under the review for more changes_.

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @SimonGideon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing success ✔️
The project is running successfully ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**